CHANGELOG.md

```diff
@@ -1,14 +1,16 @@
 # Unreleased
 
 - **[FIX]** Check icon animation not played correctly on Safari
```

Comment:
Can you move it in the `# v0.22.0` section ?

Reply:
It was not rolled out in the V0.22